README.md

```diff
@@ -378,6 +378,27 @@ After starting image service, users can open it via http://<image-server-ip>:707
 
 Unit test coverage report can be found at ./coverage
 
+### Run smoke test with FIT
```

Comment:
Could it be better if we  describe like this:
One Approach To Run Image Service API Related Case in FIT From Scratch 

As dev can also start image-service without docker,  and maybe he's already  got the latest code